site/app/templates/grading/electronic/Details.twig

```diff
@@ -27,6 +27,24 @@
     {# /Import+Export teams buttons #}
 
     <br />
+    <div id="legend">
```

Comment:
I recommend using something more descriptive than "legend" maybe "details-legend" or something like that?
You can also reuse the CSS class ".table-bordered" in your div and this will create the same border effect you added.